en/core-utility-libraries/email.rst

```diff
@@ -166,8 +171,6 @@ The following configuration keys are used:
 - ``'attachments'``: List of files to attach. See ``CakeEmail::attachments()``.
 - ``'emailFormat'``: Format of email (html, text or both). See ``CakeEmail::emailFormat()``.
 - ``'transport'``: Transport name. See ``CakeEmail::transport()``.
-- ``'log'``: Log level to log the email headers and message. ``true`` will use
-  LOG_DEBUG. See also ``CakeLog::write()``
```

Comment:
Just my two cents, but if it is going to remain in this section I think it needs a better explanation since you can't set it the same way you set all of the other config options listed in this section.  It can be confusing since there are no examples showing how one would set the log level otherwise.

As an alternative, the log method could be added to the CakeEmail class so it can be set the same way the others config options can be?